### PR TITLE
Tabletbalancer docs

### PR DESCRIPTION
**Pull request by demmer**
Add website command line and feature doc for https://github.com/vitessio/vitess/pull/16351

### CODE REVIEW

File: content/en/docs/21.0/reference/features/tablet-balancer.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: VTGate Tablet Balancer
+aliases: ['/docs/user-guides/tablet-balancer/','/docs/reference/tablet-balancer/']
```
Comment (deepthi) [inline content/en/docs/21.0/reference/features/tablet-balancer.md]: Do we really need aliases for net-new docs?
@frouioui do you know?

**Review by deepthi — COMMENTED** [review]
LGTM
The only question I have is whether this should be marked as experimental. That gives us the freedom the change flags etc. if we need to.
@demmer @rohit-nayak-ps @systay WDYT?

File: content/en/docs/21.0/reference/features/tablet-balancer.md (hunk 1 above)
Comment (demmer) [reply content/en/docs/21.0/reference/features/tablet-balancer.md]: FWIW This was totally copy-pasted on my end.

Comment (deepthi) [reply content/en/docs/21.0/reference/features/tablet-balancer.md]: Then this line should be deleted.

Comment (demmer) [reply content/en/docs/21.0/reference/features/tablet-balancer.md]: OK done.

File: content/en/docs/21.0/reference/programs/vtgate/_index.md [hunk 2]
```diff
@@ -45,6 +45,9 @@ vtgate \
       --allow-kill-statement                                             Allows the execution of kill statement
       --allowed_tablet_types strings                                     Specifies the tablet types this vtgate is allowed to route queries to. Should be provided as a comma-separated set of tablet types.
       --alsologtostderr                                                  log to standard error as well as files
+      --balancer-enabled                                                 Whether to enable the tablet balancer to evenly spread query load
```
Comment (venkatraju) [inline content/en/docs/21.0/reference/programs/vtgate/_index.md]: Minor change to the flag name in the feature PR. This should now be `--enable-balancer                                                 Enable the tablet balancer to evenly spread query load for a given tablet type` and moved down to be above `--enable-partial-keyspace-migration`

Comment (deepthi) [reply content/en/docs/21.0/reference/programs/vtgate/_index.md]: I can approve and merge once this is fixed.

**Comment by demmer** [comment]
Done. Thanks!

**Review by deepthi — APPROVED** [review]

**Pull request merged**